data/fetch_recipe.py

```diff
@@ -7,35 +7,39 @@
 
 logger.debug("Running fetch_recipe.py...")
 
-API = "https://terraria.wiki.gg//api.php"
-PARAMS = {
-    "action": "cargoquery",
-    "format": "json",
-    "tables": "Recipes",
-    "fields": "result, resultid, station, args",
-    "offset": 0,
-}
-
-S = requests.Session()
-R = S.get(url=API, params=PARAMS)
-DATA = R.json()
-final = []
-
-while DATA["cargoquery"] != []:
-    try:
-        for item in DATA["cargoquery"]:
-            final.append(item["title"])
+# API = "https://terraria.wiki.gg//api.php"
+# PARAMS = {
+#     "action": "cargoquery",
+#     "format": "json",
+#     "tables": "Recipes",
+#     "fields": "result, resultid, station, args",
+#     "offset": 0,
+# }
```

Comment:
I'd just remove the commented code. It has been added in previous commits so if we need to rollback that's possible